twa

```diff
@@ -57,21 +57,20 @@ function fetch_respcode {
 }
 
 function get_header {
-  header="${1}"
+  header="${1,,}"
 
   verbose "Extracting ${header}"
 
-  awk "BEGIN { IGNORECASE = 1 }
-       /^${header}:/ { print substr(\$0, index(\$0, \$2)) }" | tr -d '\r\n'
+  awk "tolower(\$0) ~ /^${header}:/ { print substr(\$0, index(\$0, \$2)) }" | tr -d '\r\n'
 }
 
 function get_field {
-  field="${1}"
+  field="${1,,}"
```

Comment:
Same thing with this line as well.
`field="$(echo "${1}" | tr '[:upper:]' '[:lower:]')"`